firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionLifecycleService.kt

```diff
@@ -83,6 +101,7 @@ internal class SessionLifecycleService : Service() {
         }
       }
       lastMsgTimeMs = msg.getWhen()
+      updateSessionStorage(lastMsgTimeMs) // TODO(rothbutter) don't think we need this, eh?
```

Comment:
Not sure. If we don't, that will be much better. I am a bit worried about using a timestamp like this, because clocks can change with daylight savings and leap seconds and whatever. In the `WallClock` object there is `elapsedRealtime` which is a monotonic increasing clock so won't  go backwards no matter what happens on the calendar.  Otherwise an hour forward might make everything a new session.